README.md

```diff
@@ -36,7 +36,12 @@ var hermes = require('runnable-hermes').hermesSingletonFactory({
 
 var jobCallback = function (data, done) {
   // perform operation w/ data
-  done(); // remove job from RabbitMQ queue
+  done(); // remove job from RabbitMQ queue (ack)
+
+  // optionally
+    // reject = true - negative acknowledge job completion
+    // requeue = true - requeue job and resubmit to worker
+  done(/* reject, requeue */);
```

Comment:
Another option would be consider accepting an error to done - and just following what you think would be the most used behavior (reject and requeue upon receiving an error?)